### ISSUE TITLE
Forms Share menu accessibility

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#2236]
The Share menu link in the dropdown/menu item is clickable but not keyboard navigable and would fail [WCAG 2.1 AA SC (Success Criteria) 2.1.1](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=211#keyboard-accessible).

The fix is to add keyboard navigation to the sub menu item.

**Comment by thiessenp-cds** [comment]
This issue has been resolved. It was fixed by making the share menu keyboard accessible (woho!)
